cloudbuild_cosign.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/busybox/sh
```

Comment:
Will this work properly on GCB?

Reply:
yup, tested on my on project

it fixes this error:

```
Step #0: sh: ./cloudbuild_cosign.sh: not found
```